R/shiny_models_multicls.R

```diff
@@ -120,30 +129,45 @@ shiny_models.multi_cls_shiny_data <-
       )
     )
 
+    # Define server logic required to draw a histogram
     server <- function(input, output) {
-      selected_rows <- shiny::reactiveVal()
+      # a df with all the predictions across all models; for Carson's code, model = .config and id = .row
+      selected_ids <- shiny::reactiveVal()
```

Comment:
I think that I committed code that changes this to `selected_config`. Keep it consistent between the modules.